### ISSUE TITLE
What is the range for the seed value?

### ISSUE THREAD

**Issue by SamitM1** [genmoai/models#28]

**Comment by tongnews** [comment]
Agree. I think the seed is within 2^32-1. I encountered the same issue, the weights provided seems lack diversity and the result is quite different from the online version. Would like to know the reason.

**Comment by ved-genmo** [comment]
Online is using an autoprompter. (This model does much better with LLM "upsampling" of short prompts). We don't use a negative prompt on the website, it should be unnecessary, although it seems like it can help in some cases.

Tbh, I don't know what the range is of the seed value. Presumably it's whatever can seed python's random, numpy, and torch at the same time.

**Comment by SamitM1** [comment]
Thanks for your response @ved-genmo 

But have tried enhancing the prompt with gpt4o but still the video quality is nothing compared to what is produced on the website. 
Would it be possible to provide more info about the LLM "upsampling" like what prompt your passing in to the LLM or any other code that is enhancing the original user prompt?

**Comment by SamitM1** [comment]
Furthermore there are cases where the model produces some good videos but its not as consistent as the website. Based on some experiments we have done, the website produces good quality videos fairly consistently but when we run the model ourselves the videos have a lot of morphing subjects and inconsistent faces or backgrounds.  Also some of the videos are just not realistic and cartoons even after repeatedly mentioning "realistic", "the video is hyper realistic," "the dragon is realistic."

We would greatly appreciate the code and prompt for the LLM upsampling! Thanks.

**Comment by ptits** [comment]
may be web version just uses 720p model?

**Comment by yan-genmo** [comment]
Hi @SamitM1, even if you enhance the prompt with GPT-4o, the prompt won't be the same as the autoprompting result on our website. Prompting is very important as our model is not trained on all types of captions. We will provide some help on prompting later.
For warping and morphing, you can also try increase the `cfg_scale` parameter. We use `cfg_scale=6` on the website higher than the current default of 4.5 in the repo.

**Comment by SamitM1** [comment]
@yan-genmo Thanks!

What upscaller do you guys use on the website? Becuase right now it looks like the resolution is 2x the 480p resolution.

**Comment by ptits** [comment]
> @yan-genmo Thanks!
> 
> What upscaller do you guys use on the website? Becuase right now it looks like the resolution is 2x the 480p resolution.

very good question!

**Comment by weathon** [comment]
I am also experencing very bad quality with local models. Could you provide the negative prompt or positive prompt guide?
Thanks
